literals/other.go

```diff
@@ -0,0 +1,27 @@
+package literals
```

Comment:
please don't name files "other" or "util". If you don't know what to name this, just join these lines with the main file.